src/Panel/View.php

```diff
@@ -262,7 +262,7 @@ public static function globals(): array
 		return [
 			'$config' => fn () => [
 				'api'         => [
-					'methodOverwrite' => $kirby->option('api.methodOverwrite', true)
+					'methodOverride' => $kirby->option('api.methodOverride', true)
```

Comment:
`Suggestion` to keep backwards-compatibility:

```suggestion
					// TODO: Remove fallback to `api.methodOverwrite` in v6
					'methodOverride' => $kirby->option('api.methodOverride', $kirby->option('api.methodOverwrite', true))
```

We could add "Deprecated: Option `api.methodOverwrite`: Use `api.methodOverride` instead" into the release notes. I'd also be fine with the breaking change though as it's unlikely many users would be affected.

Reply:
I'll let Basti decide on it. Totally valid suggestion. My take here was that it's gonna affect so few to none that we might just get it over right with v5.

Reply:
I would agree with @distantnative here. It seems like one of the really minor breaking changes to me for v5 and in return we get a cleaner setup.